pytubefix/__main__.py

```diff
@@ -58,6 +58,7 @@ def __init__(
             use_oauth: bool = False,
             allow_oauth_cache: bool = True,
             token_file: Optional[str] = None,
+            outh_verifier: Callable[[str, str], None]|None=None
```

Comment:
Hello, would you mind adding this parameter to the `Playlist`, `Channel` and `Search` classes to be in line with #187.

Reply:
not today, but yeah sure

Reply:
I'm going to create an oauth_verifier branch, let me work on it until it's ready

Reply:
So, you will make those changes in Playlist, Channel and Search?

Reply:
Yes, I've already added it, check if it's correct and test to see if it's working correctly, I won't be able to test it now, https://github.com/JuanBindez/pytubefix/commit/397fd8c374a15d6b05465e3a8f41b07f2f567c9e

Reply:
it's working fine

Reply:
I haven't been able to test it yet, but it seems to me that the oauth_verifier was missing in the Channel extractors IDs and in the Playlist video_generator.

Reply:
true, I'll put videos_generator

Reply:
> I haven't been able to test it yet, but it seems to me that the oauth_verifier was missing in the Channel extractors IDs and in the Playlist video_generator.

now oauth_verifier has been added to the ID extractors https://github.com/JuanBindez/pytubefix/commit/fa8eaaeba63d20e5cbd31052c83a367ab7901fb4 see if it is correct and test

Reply:
it's working fine.

Maybe this will close #165.

Reply:
This is very good, I will release a test version

Reply:
Yes, now it's easy to handle oauth (specially in UI related apps), earlier it was a headache.